### ISSUE TITLE
Testing onionify readonly component with addPrevState

### ISSUE THREAD

**Issue by nlarche** [jvanbruegge/cyclejs-test-helpers#1]
It's not possible to use the addPrevState method with a cyclejs component which is only reading the state and which isn't returning onion sinks.

I think this method might be updated with something like this :

```js
export function addPrevState(main : any, prevState : any) : any {
    return function(sources : any) : any {
        const initReducer = xs.of(() => prevState);
        const appSinks = main(sources);
        return {
            ...appSinks,
            onion: xs.merge(initReducer, appSinks.onion || xs.empty())
        };
    };
}
```
Or we might create another method to test readonly component ?

what do you think about this , I can make a PR, if you need ?

**Comment by jvanbruegge** [comment]
yeah, look good, please make a PR. But I would use `xs.never()`

**Comment by jvanbruegge** [comment]
Published in 1.5.2
